pixi/src/ui/I18n.js

```diff
@@ -26,12 +26,18 @@ class I18n {
   }
 
   async init() {
-    const pixiContent = await Promise.all([
-      AMP.getState('pixi.i18n'),
-      AMP.getState('pixiStatusBanners'),
-      AMP.getState('pixiRecommendations'),
-      AMP.getState('pixiInfoTexts'),
-    ]);
+    let pixiContent = [];
+    try {
+      pixiContent = await Promise.all([
+        AMP.getState('pixi.i18n'),
+        AMP.getState('pixiStatusBanners'),
+        AMP.getState('pixiRecommendations'),
+        AMP.getState('pixiInfoTexts'),
+      ]);
+    } catch (e) {
```

Comment:
This will help nothing, since two lines later we will access the pixi state elements that will not be there.
You should write a unit test for I18n, where you can produce all the different error cases and test that the API will handle them gracefully.

Reply:
Also when the init function handles all error cases, the question is what will happen later when the get... functions are called and the expected content is not there?

I think it makes more sense to handle errors at a higher level and show a static status banner text with some "internal error" message.
Of course we have to ensure that all errors that cannot really be handled bubble up.
We have to double check the error handling in all those async functions.

Reply:
I could have bet that you are going to comment this... these are valid points but my intention for now is just to track down if that's really the problem with #4458.

Created https://github.com/ampproject/amp.dev/issues/4604 for further discussion, I am not intending to fix this in this PR.

Reply:
Would have been nice to know the intention of this PR right from the start ;-)

Reply:
Then there would have been no bet to win 😁 Thanks though for taking a look!